tap-agent/src/agent/sender_account.rs

```diff
@@ -574,6 +575,19 @@ impl Actor for SenderAccount {
 
                 match receipt_fees {
                     ReceiptFees::NewReceipt(value) => {
+                        // If state is denied and receivied new receipt, sender was removed manually from DB
+                        if state.denied {
+                            tracing::warn!(
+                                "
+                                No new receipts should have been received, sender has been denied before. \
+                                You SHOULD NOT remove a denied sender manually from the database. \
+                                If you do so you are exposing yourself to potentially LOOSING ALL of your query
+                                fee MONEY.
```

Comment:
I liked the messages, do you think some **** around the capitalized letters can make it more flashy?

Reply:
Okay ya